examples/sdk/reactNative/ios/rnbtexample.xcodeproj/project.pbxproj

```diff
@@ -525,7 +511,7 @@
 					"-lc++",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)";
-				PRODUCT_NAME = reactNative;
+				PRODUCT_NAME = rnbtexample;
```

Comment:
example name and missing soure map upload scripts

Reply:
changed the example name, and fixed sourcemap support.